pkg/discovery/dtofactory/property/pod_properties.go

```diff
@@ -109,3 +110,17 @@ func GetPodInfoFromProperty(properties []*proto.EntityDTO_EntityProperty) (strin
 
 	return podNamespace, podName, nil
 }
+
+// Add volume property to a pod's properties
+func AddVolumeProperties(properties []*proto.EntityDTO_EntityProperty) []*proto.EntityDTO_EntityProperty {
+	volumePropertyNamespace := VCTagsPropertyNamespace
+	volumePropertyName := k8sVolumeAttached
+	volumePropertyValue := strconv.FormatBool(true)
```

Comment:
You can actually simply set this to string "true".

Reply:
done